server/src/main/scala/org/http4s/server/armeria/ArmeriaServerBuilder.scala

```diff
@@ -0,0 +1,334 @@
+/*
+ * Copyright 2013-2020 http4s.org
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.http4s
+package server
+package armeria
+
+import cats.effect.{ConcurrentEffect, Resource}
+import com.linecorp.armeria.common.util.Version
+import com.linecorp.armeria.common.{HttpRequest, HttpResponse, SessionProtocol}
+import com.linecorp.armeria.server.{
+  HttpService,
+  ServerListenerAdapter,
+  ServiceRequestContext,
+  Server => BackendServer,
+  ServerBuilder => ArmeriaBuilder
+}
+import io.micrometer.core.instrument.MeterRegistry
+import io.netty.channel.ChannelOption
+import io.netty.handler.ssl.SslContextBuilder
+import java.io.{File, InputStream}
+import java.net.InetSocketAddress
+import java.security.PrivateKey
+import java.security.cert.X509Certificate
+import java.util.function.{Function => JFunction}
+import javax.net.ssl.KeyManagerFactory
+import org.http4s.server.defaults.{IdleTimeout, ResponseTimeout, ShutdownTimeout}
+import org.http4s.syntax.all._
+import org.log4s.{Logger, getLogger}
+import scala.collection.immutable
+import scala.concurrent.duration.Duration
+
+sealed class ArmeriaServerBuilder[F[_]] private (
+    armeriaServerBuilder: ArmeriaBuilder,
+    socketAddress: InetSocketAddress,
+    serviceErrorHandler: ServiceErrorHandler[F],
+    banner: Seq[String]
```

Comment:
Unqualified `Seq` is different between 2.12 and 2.13.  You might want to make this explicitly `immutable.Seq`.  (If I were sitting down to do it today, I might just make it a `List`.  That'd be fine here, too.)